### PR TITLE
Revisions

### PR DESCRIPTION
**Pull request by adamnitti**
Trying to create a pull request for you to review my code tweaks so far. Let me know if I did this right, thanks!

### CODE REVIEW

**Comment by embryCODE** [comment]
Yeah! This is right! I'll get to this later today. I'm going to close those other PRs that I did because it's strange that I did them in the first place! Haha.

File: components/CollectionComponent.js [hunk 1]
```diff
@@ -53,7 +45,7 @@ const Collection = ({ navigation }) => {
     };
 
     // Close Add Guitar Modal
-    const onCloseModal = (state) => {
+    const toggleModal = (state) => {
```
Comment (embryCODE) [inline components/CollectionComponent.js]: The convention here is to call the param either `bool` or `isOpen` to be clear that it represents a true/false state.

**Review by embryCODE — APPROVED** [review]
Nice! Very good cleanup. I left a couple of comments. In the real world, either I would "Request changes" in this review, "Approve", or just "Comment". Just now I "Approved" and gave a couple of suggestions. That means that after you do the suggestions you can just merge without re-requesting a review. If I had requested changes then there's usual a re-request. Make sense?

**Comment by embryCODE** [comment]
So, after addressing the feedback, you merge this, delete the branch (because it's now merged to `main). Then, you make a new branch off of `main` on your machine (after pulling of course). Write more code, rinse, repeat. Cool stuff man!

**Comment by adamnitti** [comment]
Great! So I merged the request and deleted the branch from here. However I still am operating on the ### revisions branch in visual studio code. Do I then delete that local branch from visual code?

**Comment by adamnitti** [comment]
Hey, also I'm seeing that if I checkout the ### main branch in VC, it's not showing some of the changes. For example, where I replaced the shared subfolder with fixtures, fixtures is not showing up in the main branch... I'm confused.

**Pull request merged**